custom_components/afvalbeheer/sensor.py

```diff
@@ -1438,16 +1451,17 @@ def __set_picture(self, collection):
 
 class WasteDateSensor(Entity):
 
-    def __init__(self, data, waste_types, waste_collector, date_delta, dutch_days, name, name_prefix):
+    def __init__(self, data, waste_types, waste_collector, date_delta, dutch_days, name, name_prefix, language):
         self.data = data
         self.waste_types = waste_types
         self.waste_collector = waste_collector
         self.date_delta = date_delta
         self.dutch_days = dutch_days
+        self.language = language
         if date_delta.days == 0:
-            day = 'vandaag'
+            day = 'today'
```

Comment:
We can't change these, we should use a translation dict depending on language

Reply:
same for `tomorrow`

Reply:
> We can't change these, we should use a translation dict depending on language

It would be cleaner, indeed.